stickyuploads/storage.py

```diff
@@ -12,4 +15,18 @@ def __init__(self):
 
     def url(self, name):
         """Files are not accessable via url."""
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def get_available_name(self, name, max_length=None):
+        """Return path to name placed in two random directories"""
+        name = os.path.join(
```

Comment:
Probably from security stand point is better to use `mkdtemp`. If in future someone express problems with speed because of too many directories then another solution can be looked for.

I will change the logic in my PR to use `mkdtemp`.